lib/Serverless.js

```diff
@@ -70,6 +71,12 @@ class Serverless {
       return this.pluginManager.run(this.processedInput.commands);
     }
 
+    // check if tracking is enabled (and track if it's enabled)
+    const serverlessPath = this.config.serverlessPath;
```

Comment:
FYI `this.config.serverlessPath` points to the Serverless lib folder, regardless where that is, or where Serverless is installed. Check this line:
https://github.com/serverless/serverless/blob/v1.0/lib/classes/Config.js#L10